mods/persistence/modules/world_save/wrappers/_wrapper.dm

```diff
@@ -1,9 +1,11 @@
 /datum/wrapper
 	var/key
 	var/wrapper_for
+	var/deserialized_thing // the thing that this wrapper has already deserialized, if a new instance is not desired for every reference to the wrapper.
```

Comment:
This is scoped incorrectly and should probably just only be on the area wrapper, since this isn't generically assigned to at the /datum/wrapper level.